site/en/blog/extension-news-july-2023/index.md

```diff
@@ -0,0 +1,170 @@
+
+## Documentation upgrades and more Manifest V3 guidance {: #new-docs }
+
+We've also been working hard to improve the developer's learning experience. Big thanks to all of
+you who took the time to ask questions on the [chromium-group][chromium-group] and report
+documentation issues on [developer.chrome.com][github-dcc].
+
+### Highlights {: #doc-highlights }
+
+- The new [MV3 Migration][mv3-migration] section provides practical ways to convert Manifest V2 extensions to Manifest V3.
+- The [Extension service workers][sw-explainer] guide provides detailed information on extension service workers topics. These include how they are registered and updated, what the lifecycle looks like, how imports work, and more.
+- The [Handle events with service workers][sw-tut] tutorial teaches the basics of extension service workers. It builds an omnibox extension that gives you quick access to extension API reference pages.
+
+### More updates {: #doc-others }
+
+- [Using Google Analytics 4][tut-ga4] demonstrates how to track the usage of your extension popup and service worker events.
+- [Using geolocation][tut-geo] shows how to obtain the geographical location of the extension using the Offscreen API.
+- [Audio recording and screen capture][tut-capture] teaches how to capture audio and video from tabs, windows, or screens using `chrome.tabCapture` and `navigator.mediaDevices.getDisplayMedia()` APIs.
+- We added new debugging tips to the [Debugging extensions][tut-debug] guide.
+- We've updated the [Permission Warnings Guidelines][guide-perm-warn] to make it easier to understand how permission warnings work and how you can provide a better user experience. Also, there are practical ways to check what warnings the user will see.
+- Our team and contributors have also added new Manifest V3 extension samples: [WASM in extensions][gh-wasm], [Scripting API demo][gh-scripting], [Side Panel API cookbooks][gh-sidepanel], and [DeclarativeNetRequest API samples][gh-dnr]. You can explore other extension samples in our [GitHub samples repo][gh-ext-samples].
+
+### Coming soon... {: #coming-soon-docs }
+
+- How to migrate remote hosted code to Manifest V3.
+- How to run automated tests for Chrome extensions.
+- Improved Declarative Net Request guidance.
+- Improvements to the content script explainer.
+
+## 💡 Did you know? {: #tips }
+
+Before we wrap up, I wanted to share a couple of useful tools and insights:
+
+- In Chrome 115, you can access devices via [WebHID from your Chrome extension][gh-webhid].
+- [Puppeteer][puppeteer] now supports testing in headless mode with [Chrome's headless "new"][chrome-headless].
+- With the [Extension Update Testing Tool][gh-update-tool], you can check what permission warnings are triggered when permissions change in the manifest. This way, you can experience the update process as a user would. This is important because some permissions might disable the extension until the user grants access again.
```

Comment:
```suggestion
- With the [Extension Update Testing Tool][gh-update-tool], you can check what warnings are triggered when permissions change in the manifest. This way, you can experience the update process as a user would. This is important because some permissions might disable the extension until the user grants access again.
```